hugo/layouts/lookout/single.html

```diff
@@ -19,8 +19,8 @@ <h1 class="title text-left p-0 mt-5 mb-4">Empower your Developers with source{d}
             <h2 class="text-dark mb-5">source{d} Lookout</h2>
         </header>
         <div class="row text-dark mb-5">
-            <div class="col-sm-10 offset-md-1">
-                <p class="text-large mb-5">source{d} is the company driving the Machine Learning on Code (<b>#MLonCode</b>) movement, an emerging and exciting domain of research between Machine Learning, Natural Language Processing, Social Science and Programming. Doing Machine Learning on Code consists of applying ML techniques to train models that can cluster, identify and predict useful aspects of source code and software repositories.</p>
+            <div class="col-sm-12">
+                <p class="text-large mb-5">source{d} believes in the <b>Machine Learning on Code</b> (<a href="https://twitter.com/hashtag/MLonCode?src=hash">#MLonCode</a>) movement, an emerging and exciting domain of research between Machine Learning, Natural Language Processing, Social Science and Programming. Doing Machine Learning on Code consists of applying ML techniques to train models that can cluster, identify and predict useful aspects of source code and software repositories.</p>
```

Comment:
The link appears in green; used for source{d} Engine; is it ok @ricardobaeta ?
![image](https://user-images.githubusercontent.com/2437584/53827522-a13a7180-3f7b-11e9-8512-35c94dd455b4.png)

Reply:
Yes it should @dpordomingo. I'm assuming for this be tackled with ease, so LGTM.